### PR TITLE
[Refactor] reflect berry's hifi #171

### LINKED ISSUE #171: [Refactor] reflect berry's hifi
# Structure
> Be specific and detailed so that your team members can understand. Please assign the person in charge as well.

- Current structure
   - InterestEquipment UI is just low-fi version

- Better structure
   - reflect berry's hifi

- better UI

### PR DESCRIPTION
**Pull request by rriver2**
# Issue Number
🔒 Close #171

## Changes

- Changed files
campair/campair/Application/Assets.xcassets/interestEquipmentImage.imageset/Contents.json
campair/campair/Application/Assets.xcassets/interestEquipmentImage.imageset/interestEquipmentImage.svg
campair/campair/Application/Assets.xcassets/squirrel.imageset/Contents.json
campair/campair/Application/Assets.xcassets/squirrel.imageset/img_sq.svg
campair/campair/Utility/ViewExtension.swift
campair/campair/View/Dictionary/Content/DictionaryContentView.swift
campair/campair/View/Dictionary/Main/DictionaryMainView.swift
campair/campair/View/Editor/Detail/OpeningEditorView.swift
campair/campair/View/Editor/Detail/RecommendedEquipmentCardView.swift
campair/campair/View/Editor/Main/EditorMainView.swift
campair/campair/View/InterestEquipment/InterestEquipmentView.swift
campair/campair/View/TabbarView.swift
campair/campair/ViewModel/InterestEquipment/InterestEquipmentViewModel.swift

## New features

- reflect berry's hifi

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

File: campair/campair/ViewModel/InterestEquipment/InterestEquipmentViewModel.swift [hunk 1]
```diff
@@ -9,7 +9,11 @@ import Foundation
 
 final class InterestEquipmentViewModel: ObservableObject {
     /// interestEquipments - key : name , value : ImageName
-    @Published var interestEquipments: [String: String] = [:]
+    @Published var interestEquipments: [String: String] = [:] {
+        didSet {
+            viewAppeared()
+        }
+    }
```
Comment (Gobans) [inline campair/campair/ViewModel/InterestEquipment/InterestEquipmentViewModel.swift:16]: Is there any other reason that user didSet? (in a different way than before)

**Review by Gobans — APPROVED** [review]
Good river! Sorry for my late review.

File: campair/campair/ViewModel/InterestEquipment/InterestEquipmentViewModel.swift (hunk 1 above)
Comment (rriver2) [reply campair/campair/ViewModel/InterestEquipment/InterestEquipmentViewModel.swift]: I tried coding in a different way while dealing with userdefault. 
Smile left a code review to separate the role of the use case and view model, but it didn't reflect it. 
I just took that part out of the stash and committed it.

File: campair/campair/View/InterestEquipment/InterestEquipmentView.swift [hunk 2]
```diff
@@ -16,21 +16,22 @@ struct InterestEquipmentView: View {
...
+                    VStack(spacing: 0) {
+                        Image(uiImage: UIImage(named: "squirrel")!)
+                        Text("텅 비어있습니다")
+                            .frame(maxWidth: .infinity)
+                            .padding(.top, 25)
+                    }
+                    .padding(.horizontal, 20)
+                    Spacer()
+                } else {
+                    let sortedArray = viewModel.interestEquipments.sorted { $0.0 < $1.0 }
```
Comment (sustainable-git) [inline campair/campair/View/InterestEquipment/InterestEquipmentView.swift:33]: i think `sortedArray` can be a part of viewModel

File: campair/campair/View/Editor/Detail/RecommendedEquipmentCardView.swift [hunk 3]
```diff
@@ -46,7 +46,8 @@ struct RecommendedEquipmentCardView: View {
                     self.heartButtonClicked()
                 } label: {
                     Image(systemName: viewModel.isHearted(equipmentName: equipment.name) ? "heart.fill" : "heart")
-                        .font(.title3)
+                        .font(.system(size: 18))
```
Comment (sustainable-git) [inline campair/campair/View/Editor/Detail/RecommendedEquipmentCardView.swift:49]: i think `.title3` is better in aspect of accessibility?
any reason?

**Review by sustainable-git — APPROVED** [review]
LGTM

File: campair/campair/View/Editor/Detail/RecommendedEquipmentCardView.swift (hunk 3 above)
Comment (rriver2) [reply campair/campair/View/Editor/Detail/RecommendedEquipmentCardView.swift:49]: berry want to fix this image ! I just follow her decision

File: campair/campair/View/InterestEquipment/InterestEquipmentView.swift (hunk 2 above)
Comment (rriver2) [reply campair/campair/View/InterestEquipment/InterestEquipmentView.swift:33]: interestEquipments is dictionary so I sorted this fromㄱ  to ㅎ. 
And I think this is showing part so I decided to put this sorted func in here.

**Pull request merged**